composer/trainer/trainer.py

```diff
@@ -151,9 +153,18 @@ def __init__(
             log_destinations: Optional[Sequence[BaseLoggerBackend]] = None,
             callbacks: Sequence[Callback] = tuple(),
 
-            # Checkpoint hparams
-            checkpoint_loader: Optional[CheckpointLoaderHparams] = None,
-            checkpoint_saver: Optional[CheckpointSaverHparams] = None,
+            # load checkpoint
```

Comment:
Docstring should be updated to reflect change in in checkpointing arguments. It still describes `checkpoint_loader` and `checkpoint_saver`. (lines 103-106).

Reply:
Done!